addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -1387,6 +1387,42 @@ QUnit.module("Fields", (hooks) => {
         }
     });
 
+    /**
+     * check if property field popover closes when clicking on delete property icon.
+     */
+    QUnit.test(
+        "properties: close property popover once clicked on delete icon",
+        async function (assert) {
+            async function mockRPC(route, { method, model, kwargs }) {
+                if (["check_access_rights", "check_access_rule"].includes(method)) {
+                    return true;
+                }
+            }
+            await makeView({
+                type: "form",
+                resModel: "partner",
+                resId: 3,
+                serverData,
+                arch: `
+                <form>
+                    <sheet>
+                        <group>
+                            <field name="company_id"/>
+                            <field name="display_name"/>
+                            <field name="properties" widget="properties"/>
+                        </group>
+                    </sheet>
+                </form>`,
+                mockRPC,
+            });
+
+            // We open the property popover
+            await click(target, ".o_property_field:first-child .o_field_property_open_popover");
```

Comment:
just after this line, can we check that `.o_field_property_definition` is in the DOM ?
(so we make sure that the `assert.containsNone` is passed because the popover has been closed, and not because we renamed the class)

Reply:
Done :+1: